### PR TITLE
Fixing typo's in Gitwash.

### PR DESCRIPTION
**Pull request by HGWright**
## 🚀 Pull Request

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

### CODE REVIEW

**Review by bjlittle — CHANGES_REQUESTED** [review]
Welcome aboard @HGWright :partying_face: 

This PR looks good to me. Thanks for your first (of many :wink:) contribution to `iris`!

I think that it's totally appropriate for you to add an entry to the `whatsnew` [latest.rst](https://github.com/SciTools/iris/blob/main/docs/src/whatsnew/latest.rst). In particular,

- to the [Announcements](https://github.com/SciTools/iris/blob/main/docs/src/whatsnew/latest.rst#-announcements) section, highlighting that this is your **first** contribution (see the first entry [here](https://scitools-iris.readthedocs.io/en/v3.4.0/whatsnew/index.html#announcements) in the `iris` v3.4.0 release `whatsnew`, as an example)
- to the [Documentation](https://github.com/SciTools/iris/blob/main/docs/src/whatsnew/latest.rst#-documentation) section, detailing your change. Simply follow the style guide and format of other existing entries 

Lookin' forward to banking this PR :smile:

File: docs/src/developers_guide/gitwash/set_up_fork.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Overview
 
    git clone git@github.com:your-user-name/iris.git
    cd iris
-   git remote add upstream git://github.com/SciTools/iris.git
```
Comment (bjlittle) [inline docs/src/developers_guide/gitwash/set_up_fork.rst:18]: Nice spot :eagle: :eyes:

**Comment by bjlittle** [comment]
@HGWright As you aren't an `iris` core developer yet, you need to add and entry at the bottom of the `latest.rst` to resolve the link to our GH user i.e.,
```
.. comment
    Whatsnew author names (@github name) in alphabetical order. Note that,
    core dev names are automatically included by the common_links.inc:

.. _@HGWright: https://github.com/HGWright                     <------ add this here
```
If you don't do this, then you'll find that your user name is a broken link. Confirm this is the case by building the docs and viewing the `latest.rst` before and after you make this change.

File: docs/src/whatsnew/latest.rst [hunk 2]
```diff
@@ -94,6 +95,8 @@ This document explains the changes made to Iris for this release
    for the outstanding work of `@ed-hawkins`_ et al for `#ShowYourStripes`_.
    (:pull:`5141`)
 
+#. `@HGWright`_ fixed some typo's from Gitwash. (:pull:`5145`)
+
```
Comment (bjlittle) [inline docs/src/whatsnew/latest.rst:99]: Minor point, but we have **two** blank lines between sections to help the developers eye when writing the docs

**Comment by trexfeathers** [comment]
> @HGWright As you aren't an `iris` core developer yet, you need to add and entry at the bottom of the `latest.rst` to resolve the link to our GH user i.e.,
> 
> ```
> .. comment
>     Whatsnew author names (@github name) in alphabetical order. Note that,
>     core dev names are automatically included by the common_links.inc:
> 
> .. _@HGWright: https://github.com/HGWright                     <------ add this here
> ```
> 
> If you don't do this, then you'll find that your user name is a broken link. Confirm this is the case by building the docs and viewing the `latest.rst` before and after you make this change.

That's my bad @bjlittle! @HGWright asked me and I encouraged him to put the link into `common_links.inc`. I think we put @ESadek-MO there before he was a core dev, too. Do you reckon this method is a problem?

**Comment by bjlittle** [comment]
> That's my bad @bjlittle! @HGWright asked me and I encouraged him to put the link into `common_links.inc`. I think we put @ESadek-MO there before he was a core dev, too. Do you reckon this method is a problem?

No worries, all good. Yup, my eye glazed over that entry. I'm happy if you are. All good :smile:

**Review by bjlittle — APPROVED** [review]

**Pull request merged**